run

```diff
@@ -3,6 +3,10 @@
 import argparse
 import os
 import subprocess
+import getpass
+
+if os.name == "posix": # Assume linux/MacOs variants
```

Comment:
is this guard really that useful? `sudo` doesn't work on Window either and without importing `grp` later code will crash anyway

Reply:
Yeah good point